apps/static/src/app/app.element.ts

```diff
@@ -0,0 +1,383 @@
+import './app.element.scss';
+
+export class AppElement extends HTMLElement {
+  public static observedAttributes = [];
+
+  connectedCallback() {
+    const title = 'static';
+    this.innerHTML = `
+    <div class="wrapper">
+      <div class="container">
+        <!--  WELCOME  -->
+        <div id="welcome">
+          <h1>
+            <span> Hello there, </span>
```

Comment:
Can we clear out all this NX boilerplate code?

Reply:
I removed all references to it and just forgot to remove files itself. It's cleaned up